docs/QA_Checklist.md

```diff
@@ -0,0 +1,59 @@
+# QA Checklist
+
+## Sandbox account credentials
+
+If you are using the PayPal or Stripe sandbox accounts in [sample.env](../sample.env) then you can use the following credentials/credit cards for QA
+
+### Paypal accounts
+
+send-donation@test.com / testtest
+
+### Stripe credit cards
+
+- Visa **4242 4242 4242 4242**
+- Expiry date - Any future date ie **01/19**
+- CVC - Any 3 digit number ie **123**
+- First Name - Any name ie **Mary**
+- Last Name - Any name ie **Roe**
+- Country - Any country ie **Canada**
+- Address - Any address ie **366 Adelaide St W**
+- City - Any city ie **Toronto**
+- Zip/Postal code - Any zip/postal code ie **M5V 1R9**
+- State/Province - Any state/province ie **Ontario**
+- Email - Any email ie **maryroe@example.org**
+
+Stripe also provides many other [test credit cards](https://stripe.com/docs/testing#cards) for other brands and Stripe error messages.
+
+## Flows
+
+### PayPal flow
```

Comment:
Should we have a slightly modified flow for testing recurring donations for stripe and PayPal?